app/lib/actions/katello/capsule_content/sync.rb

```diff
@@ -32,6 +32,10 @@ def plan(smart_proxy, options = {})
 
           smart_proxy_helper = ::Katello::SmartProxyHelper.new(smart_proxy)
           repositories = smart_proxy_helper.repos_available_to_capsule(environment, content_view, repository)
+          if repositories.any? { |repo| smart_proxy.pulp3_support?(repo) }
+            smart_proxy.ping_pulp3
+            plan_action(Actions::Pulp3::CapsuleContent::RefreshContentGuard, smart_proxy)
+          end
           smart_proxy.ping_pulp3 if repositories.any? { |repo| smart_proxy.pulp3_support?(repo) }
```

Comment:
Calling refreshContentGuard here so it is planned only once. The Action itself does not perform anything if there is a Content Guard configured.

Reply:
i'm not sure that we need a ping_pulp3 here, planning Actions::Pulp3::CapsuleContent::RefreshContentGuard should take care of that because it inherits from  Pulp3::AbstractAsyncTask, which inherits from Pulp3::Abstract, which includes the Actions::Middleware::Pulp3ServicesCheck check

Reply:
Oh..I meant to remove the 2nd of these ping_pulp3 lines..Let me test after removing both..

Reply:
Updated..